blt/src/Blt/Plugin/Commands/UsersCommands.php

```diff
@@ -70,7 +87,9 @@ protected function isAdminBlockValid($admin) {
       array_key_exists('username', $admin) &&
       $admin['username'] != '' &&
       array_key_exists('password', $admin) &&
-      $admin['password'] != ''
+      $admin['password'] != '' &&
+      array_key_exists('enabled', $admin) &&
+      is_bool($admin['enabled'])
```

Comment:
This condition isn't quite right yet.

The `username` and `password` keys are both required.

If the `enabled` key is **NOT** present, the block is still valid.

If the `enabled` key **IS** present, the block is only valid if the value of `enabled` is a boolean.

Break the checks into multiple steps. That should make it clearer what needs to happen.